xbmc/platform/win32/input/IRServerSuite.cpp

```diff
@@ -76,14 +76,14 @@ void CIRServerSuite::Process()
 {
   m_profileId = CServiceBroker::GetProfileManager().GetCurrentProfileId();
   m_irTranslator.Load(IRSS_MAP_FILENAME);
+  unsigned int nRetryCount(6);
```

Comment:
to low amount of tries.

Reply:
If you can make it activateable in settings I'm fine with removing the count